### PR TITLE
Api600 licenses

### PR DESCRIPTION
**Pull request by Sowmyahsg**
### Description
Support for API 600 for license

### Check List
- [X] New functionality includes testing.
  - [X] All tests pass for Python 2.7+ & 3.4+(`$ tox`).
- [X] New functionality has been documented in the README if applicable.
  - [X] New functionality has been thoroughly documented in the examples (please include helpful comments).
  - [ X] New endpoints supported are updated in the endpoints-support.md file.
- [X ] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: examples/licenses.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    "key": "Use the example key mentioned above",
+    "type": "LicenseV500"
+}
+
+# Add a License
+license = oneview_client.licenses.create(options)
+print("\n\nLicense added '%s' successfully.\n" % license['key'])
+
+# Get all licenses
+print("\n\n\n\n **********   Displaying all the Licenses loaded on DCS:  ********** \n\n\n\n")
```
Comment (madhav-bharadwaj) [inline examples/licenses.py]: 'DCS' should be removed here as the user may use it for the real hardware as well. "Displaying all the Licenses loaded on the appliance" would be apt.

File: examples/licenses.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+config = try_load_from_file(config)
+oneview_client = OneViewClient(config)
+
+# Example license Key to be used
+# ACDE B9MA H9PY CHW3 U7B5 HWW5 Y9JL KMPL B89H MZVU DXAU 2CSM GHTG L762 KQL7 EG5M KJVT D5KM AFVW TT5J F77K NXWW BPSM YF26 28JS EWTZ X36Q
+# M5G7 WZJL HH5Q L975 SNJT 288F ADT2 LK44 56UG V8MC 2K9X 7KG2 F6AD EMVA 9GEB 95Y6 XBM3 HVDY LBSS PU24 KEWY JSJC FPZC 2JJE
+# ZLAB\"24R2-02192-002 T1111A HP_OneView_w/o_iLO_Explicit_Feature J4E8IAMANEON\"
+
+options = {
+    "key": "Use the example key mentioned above",
```
Comment (madhav-bharadwaj) [inline examples/licenses.py]: Maybe something like `"key": "<your license key>"` can be mentioned, as the user may not use the same license key.

File: examples/licenses.py [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+license = oneview_client.licenses.create(options)
+print("\n\nLicense added '%s' successfully.\n" % license['key'])
+
+# Get all licenses
+print("\n\n\n\n **********   Displaying all the Licenses loaded on DCS:  ********** \n\n\n\n")
+licenses = oneview_client.licenses.get_all()
+pprint(licenses)
+
+# Get License by ID
+print("\n\n ************   Getting the ID to fetch the license by ID:  ************ \n")
```
Comment (madhav-bharadwaj) [inline examples/licenses.py]: This print statement can be avoided. 'Fetching license by ID' would be apt.

File: examples/licenses.py [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+print("\n License fetched by ID is: \n")
+license = oneview_client.licenses.get_by_id(uri)
+pprint(license)
+
+# Delete License by ID
+print ("\n\n   ********** Delete the license by ID:  **********")
+print uri
+oneview_client.licenses.delete(uri)
+
+print("\n Check if the license is Deleted: \n")
```
Comment (madhav-bharadwaj) [inline examples/licenses.py]: This check is not needed. We just need to show the usage of our endpoints in the example file and I think the above code does that..

Comment (Sowmyahsg) [reply examples/licenses.py]: retaining this check as it cannot be verified on oneview appliance directly.

**Pull request merged**